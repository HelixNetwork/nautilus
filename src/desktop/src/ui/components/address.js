import React from "react";
import classNames from "classnames";
import css from "./address.scss";
/**
 * Component to set the 
 */
export default class Address extends React.PureComponent {


render(){
  
  return  (
   <div className={(css.addressbox)}>
       <div> </div>
       
       <div>Ready</div>
   </div>
  );
};
}

