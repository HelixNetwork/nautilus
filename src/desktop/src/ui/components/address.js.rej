diff a/src/desktop/src/ui/components/address.js b/src/desktop/src/ui/components/address.js	(rejected hunks)
@@ -1,3 +1,7 @@
-import React from "react";
-import css from "./address.scss";
-import { connect } from "react-redux";
+import React from 'react';
+import css from './address.scss';
+import { connect } from 'react-redux';
+import { generateNewAddress } from 'actions/wallet';
+import Scrollbar from 'ui/components/scrollbar';
+import PropTypes from 'prop-types';
+import { withI18n } from 'react-i18next';
@@ -5,11 +9,5 @@ import {
-  generateNewAddress
-} from "actions/wallet";
-import Scrollbar from "ui/components/scrollbar";
-import PropTypes from "prop-types";
-import { withI18n } from "react-i18next";
-import {
-  selectLatestAddressFromAccountFactory,
-  selectAccountInfo,
-  getSelectedAccountName,
-  getSelectedAccountMeta
-} from "selectors/accounts";
+    selectLatestAddressFromAccountFactory,
+    selectAccountInfo,
+    getSelectedAccountName,
+    getSelectedAccountMeta,
+} from 'selectors/accounts';
@@ -17,4 +15 @@ import map from 'lodash/map';
-import {
-  mapNormalisedTransactions,
-  formatRelevantTransactions
-} from "libs/hlx/transfers";
+import { mapNormalisedTransactions, formatRelevantTransactions } from 'libs/hlx/transfers';
@@ -22 +17 @@ import {
- * Component to set the 
+ * Component to set the
@@ -25,61 +20,13 @@ class Address extends React.PureComponent {
-  static propTypes = {
-    location: PropTypes.object,
-    history: PropTypes.shape({
-      push: PropTypes.func.isRequired
-    }).isRequired,
-    /** @ignore */
-    generateNewAddress: PropTypes.func.isRequired,
-    /** @ignore */
-    isGeneratingReceiveAddress: PropTypes.bool.isRequired,
-    /** @ignore */
-    receiveAddress: PropTypes.string.isRequired,
-    /** @ignore */
-    account: PropTypes.object.isRequired,
-
-  }
-  state = {
-    currentlyPromotingBundleHash: false,
-    isRetryingFailedTransaction: false,
-    transactions: []
-  };
-  getAccountTransactions = accountData => {
-    const addresses = map(
-      accountData.addressData,
-      addressData => addressData.address
-    );
-    const transactions = mapNormalisedTransactions(
-      accountData.transactions,
-      accountData.addressData
-    );
-    return formatRelevantTransactions(transactions, addresses);
-  };
-  componentDidMount() {
-    const { ui, accountInfo } = this.props;
-    const isBusy =
-      ui.isSyncing ||
-      ui.isSendingTransfer ||
-      ui.isAttachingToTangle ||
-      ui.isTransitioning;
-    const isLoading = ui.isFetchingAccountInfo;
-    const currentlyPromotingBundleHash = ui.currentlyPromotingBundleHash;
-    const isRetryingFailedTransaction = ui.isRetryingFailedTransaction;
-    const tx = this.getAccountTransactions(accountInfo);
-    this.setState({
-      isBusy: isBusy,
-      isLoading: isLoading,
-      currentlyPromotingBundleHash: currentlyPromotingBundleHash,
-      isRetryingFailedTransaction: isRetryingFailedTransaction,
-      transactions: tx
-    });
-  }
-  listAddresses(tx){
-  }
-  render() {
-    
-      return (
-        <div className={(css.addressbox)}>
-          <Scrollbar>
-      
-          </Scrollbar>
-        </div>
-      );
+    static propTypes = {
+        location: PropTypes.object,
+        history: PropTypes.shape({
+            push: PropTypes.func.isRequired,
+        }).isRequired,
+        /** @ignore */
+        generateNewAddress: PropTypes.func.isRequired,
+        /** @ignore */
+        isGeneratingReceiveAddress: PropTypes.bool.isRequired,
+        /** @ignore */
+        receiveAddress: PropTypes.string.isRequired,
+        /** @ignore */
+        account: PropTypes.object.isRequired,
@@ -87 +34,34 @@ class Address extends React.PureComponent {
-  };
+    state = {
+        currentlyPromotingBundleHash: false,
+        isRetryingFailedTransaction: false,
+        transactions: [],
+    };
+    getAccountTransactions = (accountData) => {
+        const addresses = map(accountData.addressData, (addressData) => addressData.address);
+        const transactions = mapNormalisedTransactions(accountData.transactions, accountData.addressData);
+        return formatRelevantTransactions(transactions, addresses);
+    };
+    componentDidMount() {
+        const { ui, accountInfo } = this.props;
+        const isBusy = ui.isSyncing || ui.isSendingTransfer || ui.isAttachingToTangle || ui.isTransitioning;
+        const isLoading = ui.isFetchingAccountInfo;
+        const currentlyPromotingBundleHash = ui.currentlyPromotingBundleHash;
+        const isRetryingFailedTransaction = ui.isRetryingFailedTransaction;
+        const tx = this.getAccountTransactions(accountInfo);
+        this.setState({
+            isBusy: isBusy,
+            isLoading: isLoading,
+            currentlyPromotingBundleHash: currentlyPromotingBundleHash,
+            isRetryingFailedTransaction: isRetryingFailedTransaction,
+            transactions: tx,
+        });
+    }
+    listAddresses(tx) {}
+    render() {
+        return (
+            <div className={css.addressbox}>
+                <Scrollbar></Scrollbar>
+            </div>
+        );
+    }
+}
@@ -89,8 +69,8 @@ class Address extends React.PureComponent {
-const mapStateToProps = state => ({
-  receiveAddress: selectLatestAddressFromAccountFactory()(state),
-  isGeneratingReceiveAddress: state.ui.isGeneratingReceiveAddress,
-  hadErrorGeneratingNewAddress: state.ui.hadErrorGeneratingNewAddress,
-  account: selectAccountInfo(state),
-  accountName: getSelectedAccountName(state),
-  accountMeta: getSelectedAccountMeta(state),
-  isValidatingAddress: state.wallet.isValidatingAddress
+const mapStateToProps = (state) => ({
+    receiveAddress: selectLatestAddressFromAccountFactory()(state),
+    isGeneratingReceiveAddress: state.ui.isGeneratingReceiveAddress,
+    hadErrorGeneratingNewAddress: state.ui.hadErrorGeneratingNewAddress,
+    account: selectAccountInfo(state),
+    accountName: getSelectedAccountName(state),
+    accountMeta: getSelectedAccountMeta(state),
+    isValidatingAddress: state.wallet.isValidatingAddress,
@@ -100,2 +80 @@ const mapDispatchToProps = {
-  generateNewAddress,
-
+    generateNewAddress,
@@ -104,2 +83,2 @@ export default connect(
-  mapStateToProps,
-  mapDispatchToProps
+    mapStateToProps,
+    mapDispatchToProps,
