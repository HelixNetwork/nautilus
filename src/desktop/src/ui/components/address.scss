.addressbox{
    margin-left: 0vw;
    margin-top: 2vw;
    position: absolute;
    width: 100%;
    height: 13%;
    background-image: linear-gradient(to right, rgb(21, 35, 83),rgb(19, 33, 69));
}