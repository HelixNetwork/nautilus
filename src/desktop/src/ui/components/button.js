import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';
import { Link } from 'react-router-dom';
import css from './button.scss';

/**
 * Button component
 */
export default class Button extends React.PureComponent {
    static propTypes = {
        /** Target link */
        to: PropTypes.string,
        /** Click event callback function
         * @param {object} Event - Click event object
         * @returns {undefined}
         */
        onClick: PropTypes.func,
        /** Button content */
        children: PropTypes.node,
        /** Custom button style definitions */
        style: PropTypes.object,
        /** Is button disabled */
        disabled: PropTypes.bool,
        /** Button element type */
        type: PropTypes.oneOf(['button', 'submit']),
        /** Buttons secondary class */
        className: PropTypes.oneOf([
            'outline',
            'small',
            'outlineSmall',
            'square',
            'icon',
            'icon_hover',
            'backgroundNone',
            'navleft',
            'navright',
            'modal_navleft',
            'modal_navright',
            'log_navleft',
            'log_navright',
            'reset_button',
        ]),
        /** Is button loading state active */
        loading: PropTypes.bool,
    };

    render() {
        const { onClick, children, className, to, variant, style, type, disabled } = this.props;
        if (to) {
            return (
                <Link
                    {...this.props}
                    className={classNames(css.button, css[className], disabled ? css.disabled : null)}
                >
                    {children}
                </Link>
            );
        }
        return (
            <button
                style={style}
                type={type ? type : 'button'}
                onClick={onClick}
                disabled={disabled}
                className={classNames(css.button, css[className], css[variant])}
            >
                {children}
            </button>
        );
    }
}
