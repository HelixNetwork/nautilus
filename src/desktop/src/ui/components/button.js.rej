diff a/src/desktop/src/ui/components/button.js b/src/desktop/src/ui/components/button.js	(rejected hunks)
@@ -1,5 +1,5 @@
-import React from "react";
-import PropTypes from "prop-types";
-import classNames from "classnames";
-import { Link } from "react-router-dom";
-import css from "./button.scss";
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Link } from 'react-router-dom';
+import css from './button.scss';
@@ -11,29 +11,29 @@ export default class Button extends React.PureComponent {
-  static propTypes = {
-    to: PropTypes.string,
-    /** Click event callback function
-     * @param {object} Event - Click event object
-     * @returns {undefined}
-     */
-    onClick: PropTypes.func,
-    children: PropTypes.node,
-    style: PropTypes.object,
-    disabled: PropTypes.bool,
-    type: PropTypes.oneOf(["button", "submit"]),
-    className: PropTypes.oneOf([
-      "outline",
-      "small",
-      "outlineSmall",
-      "square",
-      "icon",
-      "icon_hover",
-      "backgroundNone",
-      "navleft",
-      "navright",
-      "modal_navleft",
-      "modal_navright",
-      "log_navleft",
-      "log_navright",
-      "reset_button"
-    ]),
-    loading: PropTypes.bool
-  };
+    static propTypes = {
+        to: PropTypes.string,
+        /** Click event callback function
+         * @param {object} Event - Click event object
+         * @returns {undefined}
+         */
+        onClick: PropTypes.func,
+        children: PropTypes.node,
+        style: PropTypes.object,
+        disabled: PropTypes.bool,
+        type: PropTypes.oneOf(['button', 'submit']),
+        className: PropTypes.oneOf([
+            'outline',
+            'small',
+            'outlineSmall',
+            'square',
+            'icon',
+            'icon_hover',
+            'backgroundNone',
+            'navleft',
+            'navright',
+            'modal_navleft',
+            'modal_navright',
+            'log_navleft',
+            'log_navright',
+            'reset_button',
+        ]),
+        loading: PropTypes.bool,
+    };
@@ -41,24 +41,23 @@ export default class Button extends React.PureComponent {
-  render() {
-    const {
-      onClick,
-      children,
-      className,
-      to,
-      variant,
-      style,
-      type,
-      disabled
-    } = this.props;
-    if (to) {
-      return (
-        <Link
-          {...this.props}
-          className={classNames(
-            css.button,
-            css[className],
-            disabled ? css.disabled : null
-          )}
-        >
-          {children}
-        </Link>
-      );
+    render() {
+        const { onClick, children, className, to, variant, style, type, disabled } = this.props;
+        if (to) {
+            return (
+                <Link
+                    {...this.props}
+                    className={classNames(css.button, css[className], disabled ? css.disabled : null)}
+                >
+                    {children}
+                </Link>
+            );
+        }
+        return (
+            <button
+                style={style}
+                type={type ? type : 'button'}
+                onClick={onClick}
+                disabled={disabled}
+                className={classNames(css.button, css[className], css[variant])}
+            >
+                {children}
+            </button>
+        );
@@ -66,12 +64,0 @@ export default class Button extends React.PureComponent {
-    return (
-      <button
-        style={style}
-        type={type ? type : "button"}
-        onClick={onClick}
-        disabled={disabled}
-        className={classNames(css.button, css[className], css[variant])}
-      >
-        {children}
-      </button>
-    );
-  }
