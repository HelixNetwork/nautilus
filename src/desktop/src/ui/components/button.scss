.button {
  position: relative;
  border: 1px solid transparent;
  border-radius: 25px;
  cursor: pointer;
  display: inline-block;
  height: 50px;
  line-height: 46px;
  letter-spacing: 0.2px;
  min-width: 200px;
  text-align: center;
  padding: 0 24px 0;
  user-select: none;
  overflow: visible;
  font-weight: 400;
  font-size: 16px;
  transition: padding 0.2s cubic-bezier(0.55, 0, 0.1, 1);
  overflow: hidden;
  &:focus {
    outline: none;
  }
  &.small,
  &.outlineSmall {
    height: 42px;
    line-height: 42px;
    min-width: 30px;
    border-radius: 5px;
    font-size: 13px;

    &.loading {
      padding: 0 12px 0 36px;
    }
  }
  &.large,
  &.inline {
    padding: 0 32px;
    line-height: 44px;
    height: 46px;
    font-size: 16px;
    text-transform: capitalize;
  }
  &:disabled,
  &.disabled {
    opacity: 0.3;
    pointer-events: none;
  }
  &.square {
    border-radius: 0px;
    font-size: 18px;
    height: 70px;
    line-height: 70px;

    &.loading {
      text-indent: -4700px;
      &:before {
        position: absolute;
        left: 50%;
      }
    }
  }
  &:before {
    content: "";
    box-sizing: border-box;
    position: absolute;
    top: 50%;
    left: 18px;
    width: 16px;
    height: 16px;
    margin: -8px 0 0 -8px;
    border-radius: 50%;
    border: 1px solid transparent;
    border-top-color: var(--color-Border);
    opacity: 0;
  }
}
.backgroundNone {
  background-color: Transparent;
  background-repeat: no-repeat;
  border: none;
  cursor: pointer;
  overflow: hidden;
  outline: none;
  color: #ffffff;
}

.navleft {
  float: left;
  font-weight: bold;
  color: #7d7c7d;
  font-size: 20px;
  transition: all 0.5s;
  text-decoration: none;
  top:2px; 
  left: 20%;
}

.modal_navleft {
  float: left;
  font-weight: bold;
  color: #7d7c7d;
  font-size: 20px;
  transition: all 0.5s;
  text-decoration: none;
  top:2px; 
  left: 0%;
}

.navleft:hover{
  span{
    color: #dcab49;
  }
  color:white;
}
.navright:hover{
  span{
    color: #dcab49;
  }
  color:white;
}

.navright {
  float: right;
  font-weight: bold;
  color: #7d7c7d;
  font-size: 20px;
  transition: all 0.5s;
  text-decoration: none;
  top:2px; 
  right: 20%;
}
.modal_navright {
  float: right;
  font-weight: bold;
  color: #7d7c7d;
  font-size: 20px;
  transition: all 0.5s;
  text-decoration: none;
  top:2px; 
  right: 0%;
}
 
.modal_navleft:hover{
  span{
    color: #dcab49;
  }
  color:white;
}
.modal_navright:hover{
  span{
    color: #dcab49;
  }
  color:white;
}

.success{
  background: green;
}
.danger{
  background: red;
}

.icon_hover {
  height: 23%;
  line-height: 100%;
  &:hover{
    color: var(--input-bg);
    transform: scale(1.3);
  }
}

.log_navleft {
  float: left;
  font-weight: bold;
  color: #7d7c7d;
  font-size: 20px;
  transition: all 0.5s;
  text-decoration: none;
  top:2px; 
  left: 20%;
  &:hover{
    color:black;
  }
}

.log_navright {
  float: right;
  font-weight: bold;
  color: #7d7c7d;
  font-size: 20px;
  transition: all 0.5s;
  text-decoration: none;
  top:2px; 
  right: 20%;
  &:hover{
    color:black;
  }
}
