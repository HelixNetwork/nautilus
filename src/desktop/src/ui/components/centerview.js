import React, { Component } from 'react'
import {Grid, Row, Col} from 'react-bootstrap';

export class centerview extends Component {
    render() {
        return (
            <div>
                
            </div>
        )
    }
}

export default centerview
