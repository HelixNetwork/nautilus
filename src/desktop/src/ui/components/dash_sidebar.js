import React from 'react';
import css from 'ui/views/wallet/wallet.scss';
import classNames from 'classnames';
import PropTypes from 'prop-types';
import { withI18n } from 'react-i18next';
import Send from 'ui/images/svg/send.svg';
import Receive from 'ui/images/svg/receive.svg';
import Chart from 'ui/images/svg/chart.svg';
import History from 'ui/images/svg/history.svg';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import { faPowerOff } from '@fortawesome/free-solid-svg-icons';
import { setAccountInfoDuringSetup } from 'actions/accounts';
import { connect } from 'react-redux';
import { getAccountNamesFromState } from 'selectors/accounts';
import { setPassword, clearWalletData } from 'actions/wallet';

/**
 * Sidebar for dashboard
 */
export class DashSidebar extends React.PureComponent {
    static propTypes = {
        location: PropTypes.object,
        t: PropTypes.func.isRequired,
        history: PropTypes.shape({
            push: PropTypes.func.isRequired,
        }).isRequired,
        accounts: PropTypes.object,
        accountNames: PropTypes.array.isRequired,
        wallet: PropTypes.object,
    };

    doLogout() {
        this.props.clearWalletData();
        this.props.setPassword({});
        this.props.history.push('/');
    }

    render() {
        const { t, active } = this.props;
        return (
            <div className={classNames(css.sidebar)}>
                <ul className={classNames(css.acco_pg)}>
                    <p style={{ marginLeft: '43px', marginBottom: '20px', marginTop: '-20px', opacity: '0.3' }}>
                        MENU <span style={{ margin: '10px' }}> v </span>
                    </p>
                    <li>
                        <div
                            id="Send"
                            onClick={() => this.props.history.push('/wallet/send')}
                            className={classNames(css.img_sr1)}
                            style={
                                active === 'send'
                                    ? { opacity: '1', borderBottom: '1px solid rgba(250, 192, 0, 0.75)' }
                                    : { opacity: '0.3' }
                            }
                        >
                            <img src={Send} alt=" " className={css.sidebar_icon} /> {t('home:send')}
                        </div>
                    </li>
                    <li>
                        <div
                            onClick={() => this.props.history.push('/wallet/receive')}
                            className={classNames(css.img_sr1)}
                            style={
                                active === 'receive'
                                    ? { opacity: '1', borderBottom: '1px solid rgba(250, 192, 0, 0.75)' }
                                    : { opacity: '0.3' }
                            }
                        >
                            <img src={Receive} alt=" " className={css.sidebar_icon} /> {t('home:receive')}
                        </div>
                    </li>

                    <li>
                        <div
                            onClick={() => this.props.history.push('/wallet/chart')}
                            className={classNames(css.img_sr1)}
                            style={
                                active === 'chart'
                                    ? { opacity: '1', borderBottom: '1px solid rgba(250, 192, 0, 0.75)' }
                                    : { opacity: '0.3' }
                            }
                        >
                            <img src={Chart} alt=" " className={css.sidebar_icon} /> {t('home:chart')}
                        </div>
                    </li>
                    <li>
                        <div
                            onClick={() => this.props.history.push('/wallet/history')}
                            className={classNames(css.img_sr1)}
                            style={
                                active === 'history'
                                    ? { opacity: '1', borderBottom: '1px solid rgba(250, 192, 0, 0.75)' }
                                    : { opacity: '0.3' }
                            }
                        >
                            <img src={History} alt=" " className={css.sidebar_icon} /> {t('home:history')}
                        </div>
                    </li>
                    <li>
                        <div
                            onClick={() => this.props.history.push('/settings/editname')}
                            className={classNames(css.img_sr1)}
                            style={{ paddingTop: '20px', opacity: '0.3' }}
                        >
                            {t('home:settings')}
                        </div>
                    </li>
                    <li>
                        <div
                            onClick={() => this.props.history.push('/wallet/support')}
                            className={classNames(css.img_sr1)}
                            style={{ opacity: '0.3' }}
                        >
                            {t('global:support')}
                        </div>
                    </li>
                    <li>
                        <div
                            onClick={this.doLogout.bind(this)}
                            className={classNames(css.img_sr1)}
                            style={{ opacity: '0.3' }}
                        >
                            <FontAwesomeIcon icon={faPowerOff} /> LOGOUT{' '}
                        </div>
                    </li>
                </ul>
            </div>
        );
    }
}

const mapStateToProps = (state) => ({
    accounts: state.accounts.accountInfo,
    accountNames: getAccountNamesFromState(state),
    wallet: state.wallet,
});

const mapDispatchToProps = {
    setAccountInfoDuringSetup,
    clearWalletData,
    setPassword,
};

export default connect(
    mapStateToProps,
    mapDispatchToProps,
)(withI18n()(DashSidebar));
