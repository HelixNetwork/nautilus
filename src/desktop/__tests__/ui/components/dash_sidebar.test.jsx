import React from 'react';
import { shallow } from 'enzyme';

import {DashSidebar}  from 'ui/components/dash_sidebar';


const props = {
        t: (str) => str,
        history:
            {
            push:  jest.fn(),
            },
        accountNames:[],
        onClick: jest.fn(),
   };

   const onClick =  jest.fn();

describe('DashSidebar component', () => {
    test('Render the DashSidebar component', () => {
        const wrapper = shallow( <DashSidebar {...props} />);

        expect(wrapper).toMatchSnapshot();
    });

    // test('Copy to clipboard event', () => {
    //     const wrapper = shallow(<DashSidebar {...props} />);

    //     wrapper.simulate('click');

    //     expect(props.history.push).toHaveBeenCalledTimes(7);
        
    // });
});