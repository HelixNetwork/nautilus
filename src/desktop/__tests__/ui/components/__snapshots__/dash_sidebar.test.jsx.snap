// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li
  page="/wallet"
>
  Send
</li>
`;

exports[`renders correctly 2`] = `
<li
  page="/wallet/send"
>
  Send
</li>
`;

exports[`renders correctly 3`] = `
<li
  page="/wallet/receive"
>
  Receive
</li>
`;

exports[`renders correctly 4`] = `
<li
  page="/wallet/history"
>
  History
</li>
`;

exports[`renders correctly 5`] = `
<li
  page="/wallet/support"
>
  Support
</li>
`;
